packages/service-library/src/servicelib/aiohttp/rest_models.py

```diff
@@ -2,13 +2,9 @@
 
 UNDER DEVELOPMENT
 """
-import typing
-import warnings
 
 import attr
 
-warnings.warn("DO NOT USE IN PRODUCTION, STILL UNDER DEVELOPMENT")
```

Comment:
Remove then above the "UNDER DEVELOPMENT" :-)

Will follow up in https://github.com/ITISFoundation/osparc-simcore/issues/2008 to completely upgrade this module

Reply:
done